index.html

```diff
@@ -1962,6 +1962,20 @@
 </p>
 </div>
 
+<div class="practice">
+<p>
+  <span class="practicelab" id="principle-fake-consent">
+    An [=actor=] should not prompt a [=person=] for consent if the
+	person is unlikely to have sufficient information to make an informed decision to consent or not.
+  </span>
+</p>
+</div>
+
+In considering whether or not a person is sufficiently informed to be promoted for consent,
+actors should be realistic in comparing [=privacy labour=] required with actual user time spent in
+the relevant context.
+Simply providing a link to a complex policy is unlikely to mean that the person is
+informed, if the policy takes half an hour to read and the average site visit is ten minutes.
```

Comment:
I'm not sure that I love this measure. I don't want to spend 30min reading a complex policy even for a site that I use every day!